app/src/main/res/values-cs/strings.xml

```diff
@@ -1,2 +1,75 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources></resources>
\ No newline at end of file
+<resources>
+    <string name="back">Zpět</string>
+    <string name="menu">Menu</string>
+    <string name="cancel">Zrušit</string>
+    <string name="add">Přidat</string>
+    <string name="share">Sdílet</string>
+    <string name="delete">Smazat</string>
+    <string name="paste">Vložit</string>
+    <string name="add_files_not_supported">Vaše zařízení nepodporuje přidávání souborů</string>
+    <string name="unknown">neznámý</string>
+    <string name="remove">Odstranit</string>
+    <string name="share_empty_state">Ještě zde nic není.</string>
+    <string name="warning_alpha_intro">Varování:</string>
+    <string name="warning_no_files_added">Nebyly vybrány žádné soubory.</string>
+    <string name="share_onion_intro"><b>Kdokoli</b> s touto OnionShare adresou může <b>stahovat</b> vaše soubory za pomoci <b>Tor Browseru</b>:</string>
+    <string name="clipboard_onion_service_label">OnionShare adresa</string>
+    <string name="clipboard_onion_service_copied">OnionShare adresa byla zkopírována do schránky.</string>
+    <string name="notification_channel_name_foreground">Status sdílení souborů</string>
+    <string name="notification_channel_name_sharing">Sdílení adresy je dostupné</string>
+    <plurals name="share_file_list_summary">
```

Comment:
This fails lint because `quantity="many"` is missing. I assume the same is true for `settings_tor_bridges_number` below.

Reply:
I set those checks as enforced now on weblate:

* [Missing plurals](https://docs.weblate.org/en/latest/user/checks.html#check-plurals)
* [Unpluralised](https://docs.weblate.org/en/latest/user/checks.html#unpluralised) 

However, `cs` is not failing any checks currently: https://hosted.weblate.org/projects/onionshare/onionshare-android/cs/#information

So maybe I didn't find the right check for this or weblate doesn't support this check?

Reply:
Perhaps there's a mismatch between the plural forms weblate considers necessary and the forms the linter considers necessary? I wonder if anything would break if we suppressed this lint rule.

Reply:
Ignoring the lint check may be an option. If `many` quantity is missing, maybe it falls back to some sane default? There's `other` and `few` here.